### ISSUE TITLE
[BUG] New placeholder doesn't display name

### ISSUE THREAD

**Issue by adelando** [Auxilor/Boosters#21]
Hello,

With the new placeholder %boosters_<name>_name% it does not display any title but seconds and other placeholders do.

for example, this is my config using TAB for bossbars...

`    booster_bonushealththree:
      style: "PROGRESS"
      color: "YELLOW"
      progress: "100"
      text: "#e7a9f5 #5ae0c8Active: #e7a9f5 %boosters_bonushealththree_name% &5&l| #5ae0c8Timer: #e7a9f5 %boosters_bonushealththree_seconds_remaining%"
      display-condition: '%boosters_bonushealththree_seconds_remaining%>1'`
      
  And this is how it displays.
![2022-05-12_16 01 11](https://user-images.githubusercontent.com/11265974/168024449-bf64a0f8-7ee1-411b-ab12-3af58b91a047.png)

**Comment by WillFP** [comment]
I can't recreate that - it definitely works for me

**Comment by adelando** [comment]
I’ve seen this not working for a few people in the boosters discord channel, all other new placeholders seem to work fine just the name placeholder. I’ve also tried shortening the id name and still the same.

**Comment by TerrrorByte** [comment]
Yeah, same for me. All the placeholders except the name placeholder work. Is there not a way to implement a placeholder to display _any_ active booster, like a wildcard? That'd be really nice

**Comment by TerrrorByte** [comment]
Actually, pretty sure I see what's going on. There isn't a name placeholder being registered by this plugin.
https://github.com/Auxilor/Boosters/blob/master/eco-core/core-plugin/src/main/kotlin/com/willfp/boosters/boosters/Booster.kt

I created a pull request, but I'm not familiar with Kotlin, so hopefully this is the correct syntax
https://github.com/Auxilor/Boosters/pull/23
